tests/unit/test.program.js

```diff
@@ -242,13 +247,34 @@ describe('program.Program', () => {
       });
   });
 
+  it('checks for updates automatically', () => {
+    const handler = spy();
+    const getVersion = () => 'some-package-version';
+    const checkForUpdates = sinon.stub();
+    const program = new Program(['run'])
+      .command('run', 'some command', handler);
+    return execProgram(program, {
+      checkForUpdates,
+      getVersion,
+      localEnv: 'production',
```

Comment:
This needs one more test that does `localEnv: 'development'` and makes sure the update checker doesn't get called, like `assert.equal(checkForUpdates.called, false)`